test/README

```diff
@@ -4,3 +4,9 @@
 The `run` script uses litani to run a variety of unit and system tests.
 The dashboard showing the output of the latest test run will always be
 symlinked from test/output/latest/html/index.html.
+
+# OPTIONS
+
+*--fast*
+	Do not run e2e tests with SLOW variable set to true
+    If the run time of a test is >=10, SLOW=True else SLOW=False
```

Comment:
Sorry to be a bit annoying, but:
> If the run time of a test is >=10, SLOW=True else SLOW=False

is missing the unit for time. Please mention:
```
If a test does not contain a boolean variable called `SLOW` that e2e will not run when this option is specified.
If the run time of an e2e test exceeds 10 seconds, then the value for the constant called SLOW should be set to True otherwise it should be set to False.
```

Reply:
I favor Nick's version as it's more laconic.

Reply:
Resolved